coops-ui/src/views/Dashboard.vue

```diff
@@ -239,13 +239,15 @@ export default {
      * Searches the filings history for a 'paid' status.
      * Paid status indicates a filing that is paid but future effective.
      * Change the state of the UI when a filing is future effective.
-     * Currently only BCOMPS have future effective filings.
+     * Currently only BCOMPs change of addresses filing are future effective.
```

Comment:
I recommend replacing all BCORP(s) with BCOMP(s) now, to avoid confusion in the code :)

Reply:
Good idea, was thinking this as well. When did we officially change from Bcorps to Bcomps.. or perhaps Bcorps was wrong all along 😆